src/widget/widgetengine.js

```diff
@@ -34,6 +35,22 @@ export default class WidgetEngine extends Plugin {
 			}
 
 			const viewSelection = conversionApi.viewSelection;
+
+			// Add CSS class if selection is placed inside nested editable that belongs to widget.
+			const editableElement = viewSelection.editableElement;
+
+			if ( editableElement && !( editableElement instanceof RootEditable ) ) {
+				const widget = editableElement.findAncestor( element => isWidget( element ) );
+
+				if ( widget ) {
+					widget.addClass( WIDGET_SELECTED_CLASS_NAME );
```

Comment:
I can see we use the ALL_CAPS notation in some cases. `INLINE_FILLER_LENGTH`, `BR_FILLER`, `NBSP_FILLER`, `DEFAULT_PRIORITY`. Also, all but one `export const` use this notation. So let's stick to that.